ruby/chat_pub.rb

```diff
@@ -0,0 +1,6 @@
+require 'ably'
```

Comment:
The JS version doesn't include necessary constructor code, so this + ably = Ably::Realtime.new(...) is not necessary. I believe this is noise, we only want to show the code that is relevant

Reply:
I see that we don't yet have finalisation of the JS version though, see https://github.com/ably/homepage-snippets/pull/1.  I appreciate this makes this job harder, but the goal should be to have as close an equivalent, that is terse.  Please can you be more terse and only show what is relevant to the visitor to Ably.  Specifically, see https://github.com/ably/homepage-snippets/pull/1#pullrequestreview-684069614, and comment:

> 1. Show how simple Ably is
> 2. Show real code that could actually works (at least the extracted part that we're showing)
> 3. Hide irrelevant boilerplate code.

Reply:
Fixed